webapp/src/containers/shareRecords/index.tsx

```diff
@@ -135,6 +137,17 @@ const ShareRecords = () => {
         return action ? handleOpenShareRecordModal : 'Share another record';
     }, [apiError]);
 
+    // Returns heading for the result panel
+    const getResultPanelHeader = () => {
+        if (apiError) {
+            return apiError.id === Constants.ApiErrorIdNotConnected || apiError.id === Constants.ApiErrorIdRefreshTokenExpired ?
+                Utils.getContentForResultPanelIfDisconnected(apiError.message, hideModal) :
+                apiError.message;
+        }
+
+        return Constants.RecordSharedMsg;
```

Comment:
```suggestion
        return apiError ? (
            apiError.id === Constants.ApiErrorIdNotConnected || apiError.id === Constants.ApiErrorIdRefreshTokenExpired ?
                Utils.getContentForResultPanelIfDisconnected(apiError.message, hideModal) :
                apiError.message;
        )
       : Constants.RecordSharedMsg;
```

see if this works, I guess nested ternary not allowed by linter

Reply:
Yes, it's not allowed. This doesn't work.